plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt

```diff
@@ -277,6 +316,19 @@ abstract class AnnotationManagerImpl<G : Geometry, T : Annotation<G>, S : Annota
     }
   }
 
+  /**
+   * Add an annotation to MapView.
+   *
+   * @param annotation the annotation added to MapView.
+   */
+  @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+  fun addAnnotation(annotation: T) {
```

Comment:
should be `internal` then?

Reply:
We need this function in test, so have to keep it as public.